### ISSUE TITLE
Tickets with Price = 0 are set to '' if validation fails on edit.

### ISSUE THREAD

**Issue by anchovy** [verbb/Events#111]
### Describe the bug

We've got some tickets that are free, ie. cost $0 each, but if an event is edited and fails validation the ticket price in the form is set to '' and subsequent saves return the error `Price cannot be blank.`

### Steps to reproduce

1. Add an event with a ticket price = 0
2. Save event
3. Edit the event & clear the title field
4. Save event
5. See the error on the ticket field:
![image](https://user-images.githubusercontent.com/599005/191127069-15684586-5338-485e-87da-7ffc3c4f13ca.png)

### Craft CMS version

Craft Pro 3.7.54

### Plugin version

1.4.22

### Multi-site?

No

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/events` requirement in `composer.json` to:

```
"require": {
  "verbb/events": "dev-craft-3 as 1.4.22",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.4.23](https://github.com/verbb/events/releases/tag/1.4.23)
